src/io/export_pov.cc

```diff
@@ -0,0 +1,122 @@
+/*
+ *  OpenSCAD (www.openscad.org)
+ *  Copyright (C) 2021      Konstantin Podsvirov <konstantin@podsvirov.pro>
+ *         Povray 2024      Folkert van Heusden  <mail@vanheusden.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  As a special exception, you have permission to link this program
+ *  with the CGAL library and distribute executables, as long as you
+ *  follow the requirements of the GNU GPL in regard to all of the
+ *  software in the executable aside from CGAL.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ */
+
+#include "export.h"
+
+#include "PolySet.h"
+#include "PolySetUtils.h"
+
+
+void export_pov(const std::shared_ptr<const Geometry>& geom, std::ostream& output, const ExportInfo& exportInfo)
+{
+  auto ps = PolySetUtils::getGeometryAsPolySet(geom);
+  if (Feature::ExperimentalPredictibleOutput.is_enabled()) {
+    ps = createSortedPolySet(*ps);
+  }
+
+  output << "// Generated by OpenSCAD!\n";
+  output << "// Source file: " << exportInfo.sourceFileName << "\n\n";
+
+  output << "#version 3.7;\n";
+  output << "global_settings { assumed_gamma 1.0 }\n";
+  output << "#declare MATERIAL=finish { specular 0.5 roughness 0.001 reflection{0 0.63 fresnel} ambient 0 diffuse 0.6 conserve_energy }\n";
+  output << "#declare MATERIAL_INT=interior{ior 1.32}\n";
+
+  auto has_color = !ps->color_indices.empty();
+
+  double avg_x = 0;
+  double avg_y = 0;
+  double avg_z = 0;
+  double tsd_x = 0;
+  double tsd_y = 0;
+  double tsd_z = 0;
+  size_t avg_n = 0;
+
+  for (size_t pi=0; pi<ps->indices.size(); pi++) {
+    const auto &t = ps->indices.at(pi);
```

Comment:
What is "t"? I assume it's short for "triangle", but in reality it's a polygon.
-> in general, try to use longer variable names, unless it's painfully obvious what the name means, as code is primarily meant for human consumption.